classes/Page/Page.mjs

```diff
@@ -0,0 +1,135 @@
+export class Page {
+    constructor(page = {}) {
+        this.page = page
+        if(!this.page.id) {
+            this.page.id = Date.now().toString();  // ticket to replace with MongoDB ObjectID()
+        }
+    }
+    
+    get id() {
+        return this.page.id
+    }
+
+    set id(id) {
+        this.page.id = id
+    }
+
+    // Method to represent the page as a canvas with annotation page layer
+    asCanvas() {
+        return {
+            type: 'Canvas',
+            '@context': 'http://iiif.io/api/presentation/3/context.json',
+            id:this.page.id ?? 0,
+            height: this.page.height ?? 0,
+            width: this.page.width ?? 0,
+            label: this.page.label ?? '',
+            summary: this.page.summary ?? '',
+            thumbnail: this.page.thumbnail ?? '',
+            navDate:this.page.navDate ?? '',
```

Comment:
some of these may be more appropriate to leave off if not present, especially if not mandatory by the IIIF spec

Reply:
Can you please be more specific on this please!!

Reply:
I had just commented out the fields , Once i have more clarity on the properties will neatly deal with it.